### DISCUSSION TITLE
Welcome to Redisson Discussions!

### DISCUSSION THREAD

**Discussion by mrniko** [redisson/redisson#4589] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Breathtak** [comment]
I have one question, CacheManager configuration Bean as follows: 
    @Bean
    public CacheManager cacheManager(RedissonClient redissonClient) {
        Map<String,CacheConfig> config = new HashMap<>();
        config.put("shop", new CacheConfig(24*60*1000, 12*60*1000));
        return new RedissonSpringCacheManager(redissonClient, config,new JsonJacksonCodec(objectMapper));

    }
Then add @Cacheable(cacheNames=''shop',key="#p0") in the service as follows:
    @Override
    @Cacheable(cacheNames = "myShop",key = "'id:'+#p0")
    public Shop queryById(Long id) {
        log.info("根据店铺id获取：{}",id);
        return getById(id);
    }
only if cacheNames is "shop" can be hit and redis has result
when change cacheNames  "myShop" as follows:
    @Override
    @Cacheable(cacheNames = "myShop",key = "'id:'+#p0")
    public Shop queryById(Long id) {
        log.info("根据店铺id获取：{}",id);
        return getById(id);
    }
The first query from database and the results are written to redis
But other query always from database 
Question  
Creating CacheManager Bean  must initialize all cacheNames in config ?
